modules/security/disk_encryption_set/keyvault_policy.tf

```
@@ -1,4 +1,5 @@
 resource "azurerm_key_vault_access_policy" "des" {
+  count        = try(var.keyvault_id, null) == null ? 0 : 1
```

Suggest you update the variable keyvault_id like

```
variable keyvault_id {
 default = null
}
```

With that variable default set to null you don't need to add the try()
```suggestion
  count        = var.keyvault_id == null ? 0 : 1
```

---

I made the change, it works :)